demos/wascc/fileserver/k8s.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fileserver-wascc
+  labels:
+    app: fileserver-wascc
+spec:
+  containers:
+    - image: webassembly.azurecr.io/fileserver-wascc:v0.1.0
+      imagePullPolicy: Always
+      name: fileserver-wascc
+      env:
+        - name: PORT
+          value: "8080"
+      ports:
+        - containerPort: 8080
+      volumeMounts:
+        - name: storage
+          mountPath: /tmp # this is ignored by waSCC, but is still necessary from the k8s API
```

Comment:
I am not clear what this means. Does that mean I can't mount `/tmp` from the host to an arbitrary directory within the container?

Reply:
No, because waSCC does not mount or pre-open directories like WASI does. It instead streams files to and from the actor using the filesystem capability.

From [the docs](https://github.com/wascc/fs-provider/):

> Because of the way WebAssembly and the waSCC host work, all `wascap:blobstore` capability providers must stream files to and from the actor. This allows actors to unblock long enough to allow other messages from other providers to be processed and keeps the WebAssembly module from allocating too much memory.

In other words, a directory is not mounted into the runtime. It's accessed through a handle to fetch and push files to the filesystem, effectively nullifying the `mountPath`.

Reply:
https://github.com/deislabs/krustlet/blob/9914e7b4f0ed611eb96b965d71d2142c6bee1406/demos/wascc/fileserver/src/lib.rs#L17-L18

Reply:
Ah ok that makes more sense